Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -463,10 +463,10 @@ private static double GetAspectRatio(ImageDataDto i)
             }
 
             StringBuilder str = new StringBuilder("[", 1 + (programIds.Count * 13));
-            foreach (ReadOnlySpan<char> i in programIds)
+            foreach (var i in programIds)
```

Comment:
This now generated a compilation error because you can't uses stack-allocated `ReadOnlySpan<char>` in an `async` method